sechub-adapter-pds/src/main/java/com/daimler/sechub/adapter/pds/PDSWebScanConfigImpl.java

```diff
@@ -82,6 +82,22 @@ public final PDSWebScanConfigBuilder setJobParameters(Map<String, String> jobPar
         @Override
         protected void customBuild(PDSWebScanConfigImpl config) {
             jobParameters.put(PDSDefaultParameterKeyConstants.PARAM_KEY_PDS_SCAN_TARGET_TYPE, config.getTargetType());
+            
+            // to get the base URl from the URL
+            // https://example.org/abc -> https://example.org
+            
+            try
+            {            
+            	URL url = new URL(PARAM_KEY_PDS_SCAN_TARGET_URL);
+				String path = url.getFile().substring(0, url.getFile().lastIndexOf('/'));
+				String baseUrl = url.getProtocol() + "://" + url.getHost() + path;
+				PARAM_KEY_PDS_SCAN_TARGET_URL = baseUrl;
+           	}
+           	catch(Exception e)
+ 			{
+ 				throw new IllegalArgumentException(PARAM_KEY_PDS_SCAN_TARGET_URL + " is not a valid URL ",e);
```

Comment:
Throwing an exception here would be a hard fail - so if a user just defines it wrong the complete job would crash and an unexpected error would be returned.

But maybe this was not clear enough specified inside the issue.